test/definitions/amount_energy_from_mass_and_temperature_test.py

```diff
@@ -19,15 +20,15 @@ def test_args():
     SI.set_quantity_scale_factor(m, 0.5 * units.liter * 1000 * units.kilogram / units.meter**3)
     t1 = units.Quantity('t1')
     SI.set_quantity_dimension(t1, units.temperature)
-    SI.set_quantity_scale_factor(t1, (323 - 273) * units.kelvin)
+    SI.set_quantity_scale_factor(t1, (273 - CELSIUS_TO_KELVIN_OFFSET) * units.kelvin)
     t2 = units.Quantity('t2')
     SI.set_quantity_dimension(t2, units.temperature)
-    SI.set_quantity_scale_factor(t2, (273 - 273) * units.kelvin)
+    SI.set_quantity_scale_factor(t2, (323 - CELSIUS_TO_KELVIN_OFFSET) * units.kelvin)
     Args = namedtuple('Args', ['C', 'm', 't1', 't2'])
     return Args(C=C, m=m, t2=t2, t1=t1)
 
 def test_basic_amount(test_args):
-    result = amount_energy.calculate_amount_energy(test_args.C, test_args.m, test_args.t1, test_args.t2)
+    result = amount_energy.calculate_amount_energy(test_args.C, test_args.m, test_args.t2, test_args.t1)
```

Comment:
Why did you change order of parameters? T1 is supposed to be initial temperature and should come first.